docs/configdoc.md

```diff
@@ -0,0 +1,104 @@
+# L3AFD Config Options Documentation
+
+See [l3afd.cfg](https://github.com/l3af-project/l3af-arch/blob/main/dev_environment/cfg/l3afd.cfg) for a full example configuration.
+
+
+```
+[DEFAULT]
+
+[l3afd]
+pid-file: ./l3afd.pid
+datacenter: dummy
+bpf-dir: /dev/shm
+bpf-log-dir:
+kernel-major-version: 4
+kernel-minor-version: 15
+shutdown-timeout: 1s
+http-client-timeout: 10s
+max-nf-restart-count: 3
+bpf-chaining-enabled: true
+swagger-api-enabled: false
+# PROD | DEV
+environment: PROD
+....
+```
+
+### Below is the detailed documentation for each field
+
+
+## [l3afd]
+| FieldName     | Example       | Description     | Required        |
+| ------------- | ------------- | --------------- | --------------- |
+|pid-file| `"./l3afd.pid"`  | The pin Path for l3afd.pid file which contains process id of eBPF Programs | Yes |
+|datacenter| `"dummy"` | Name of Datacenter| Yes |
+|bpf-dir| `"/dev/shm"` | Absolute Path of eBPF Programs (where they are extracted) | Yes |
```

Comment:
Question: is it only the ebpf program itself, or also the application that may be in the package?  If the latter, then the description here is wrong.

Reply:
The Extracted directory contain only ebpf program binary and ebpf bytecode.
There is no package